### PR TITLE
ESC-1181 - fix sector mismatch

### PR DESCRIPTION
**Pull request by masahmad**

### CODE REVIEW

File: test/uk/gov/hmrc/eusubsidycompliancefrontend/controllers/FinancialDashboardControllerSpec.scala [hunk 1]
```diff
@@ -64,6 +69,13 @@ class FinancialDashboardControllerSpec
       "play.filters.csp.nonce.enabled" -> false
     )
   )
+  override lazy val appConfig: AppConfig = app.injector.instanceOf[AppConfig]
```
Comment (habeeb01) [inline test/uk/gov/hmrc/eusubsidycompliancefrontend/controllers/FinancialDashboardControllerSpec.scala:72]: how comes these are now needed? whats different in the new test?

File: test/uk/gov/hmrc/eusubsidycompliancefrontend/test/CommonTestData.scala [hunk 2]
```diff
@@ -158,6 +158,15 @@ object CommonTestData {
     List(businessEntity4)
   )
 
+  val undertaking3 = Undertaking(
+    undertakingRef,
+    UndertakingName("TestUndertaking"),
+    agriculture,
+    IndustrySectorLimit(32.34).some,
```
Comment (WitchellJack) [inline test/uk/gov/hmrc/eusubsidycompliancefrontend/test/CommonTestData.scala:165]: One small comment which is entirely preference, for me Some(LocaDate etc) Reads better then adding .some on the end. Again preference so happy to go with what reads better for all!

**Review by WitchellJack — COMMENTED** [review]
LGTM once Habeeb's comments have been addressed, have added one small comment myself too! 👍🏻

File: test/uk/gov/hmrc/eusubsidycompliancefrontend/controllers/FinancialDashboardControllerSpec.scala [hunk 3]
```diff
@@ -106,27 +117,26 @@ class FinancialDashboardControllerSpec
...
+          subsidies = undertakingSubsidies,
+          today = fakeTimeProvider.today
+        )
+
+      status(result) shouldBe Status.OK
+      val data = contentAsString(result)
+      data shouldBe page(summaryData)(request, messages, instanceOf[AppConfig]).toString()
+      val document = Jsoup.parse(data)
+      val sectorCapElement = document.select("dt:contains(Sector cap (Agriculture))").text()
+      sectorCapElement shouldBe "Sector cap (Agriculture)"
```
[the remark this answers is not in the document]
Comment (masahmad) [reply test/uk/gov/hmrc/eusubsidycompliancefrontend/controllers/FinancialDashboardControllerSpec.scala:133]: Thanks for pointing this out  - I agree much cleaner - I've added the id in the span now

File: test/uk/gov/hmrc/eusubsidycompliancefrontend/test/CommonTestData.scala (hunk 2 above)
Comment (masahmad) [reply test/uk/gov/hmrc/eusubsidycompliancefrontend/test/CommonTestData.scala:165]: Thanks Jack , I've made the change

File: test/uk/gov/hmrc/eusubsidycompliancefrontend/controllers/FinancialDashboardControllerSpec.scala (hunk 1 above)
Comment (masahmad) [reply test/uk/gov/hmrc/eusubsidycompliancefrontend/controllers/FinancialDashboardControllerSpec.scala]: removed - not needed - copied from an example

File: test/uk/gov/hmrc/eusubsidycompliancefrontend/test/CommonTestData.scala (hunk 2 above)
Comment (WitchellJack) [reply test/uk/gov/hmrc/eusubsidycompliancefrontend/test/CommonTestData.scala:165]: nice! as long as it reads well for all thanks for changing!

**Review by habeeb01 — APPROVED** [review]
LGTM 👍

**Pull request merged**